src/lib/routes.js

```diff
@@ -3,13 +3,19 @@
 const dateFormat = 'YYYY-MM-DD HH:mm:ss';
 const moment = require('moment');
 const model = require('./model');
-const report = require('lev-react-components').LevReport;
+const dashboardModel = require('./dashboard-model');
+const { LevDashboard, LevReport } = require('lev-react-components');
 
-module.exports = server => {
-  server.get('/readiness', (req, res) => {
-    res.send('OK');
+const promiseResponder = (promise, component, req, res, next) => promise
+  .then(data => !component ? res.send(data) : res.render(component, data))
+  .catch(err => {
+    req.log.error(err);
+    return next(err);
   });
 
+module.exports = server => {
+  server.get('/readiness', (req, res) => res.end('OK'));
```

Comment:
I think this should be using `.send()`. - http://restify.com/docs/response-api/#send

Reply:
fixed